Tests/Packet++Test/Tests/SmtpTests.cpp

```diff
@@ -0,0 +1,240 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EndianPortable.h"
+#include "EthLayer.h"
+#include "IPv4Layer.h"
+#include "IPv6Layer.h"
+#include "Packet.h"
+#include "SmtpLayer.h"
+#include "SystemUtils.h"
+#include "TcpLayer.h"
+
+PTF_TEST_CASE(SmtpParsingTests)
```

Comment:
I think we can also add a username/password request packet and check that we see an unknown command